src/pages/Home/index.js

```diff
@@ -336,18 +413,32 @@ const Home = () => {
         break;
       case 'version':
         let pack_format = DEFAULT_PACK_META.pack_format;
+        newPackMeta.extension = DEFAULT_EXTENSION;
+        let showResolution = false;
         switch (event.value) {
           case '1_14':
             pack_format = 4;
+            newPackMeta.fileBuilder = javaFileBuilder;
```

Comment:
I'm a bit sceptical about this, since the `fileBuilder` isn't really part of the pack metadata. Same for `extension`. Maybe it would be better to read the `pack_format` at download-time to work out what `fileBuilder` and `extension` should be. You can just leave it if you want me to implement that instead.

Reply:
I'm leaving this as is.